docs/overview/trustbloc-did-method.md

```diff
@@ -82,63 +111,66 @@ TODO: consortium history may need to cache the stakeholder configs and verkeys -
-  - maybe we can suggest resolving stakeholder X's did doc using stakeholder Y's endpoint, to verify the stakeholders are on the same ledger?
-    - but then, if they all sign the same consortium config, which points to all of them, we don't need this complication
---> 
-### Bootstrapping Trust
-
-The Bloc DID method is designed to offer several mechanisms for a resolver to confirm that it can trust the stakeholders to provide the DID doc ledger service. 
+### Consortium Policy Configuration
+The `policy` element of a consortium config object is a JSON object. Each key-value pair is a specific rule for the client to follow when processing consortium or stakeholder configuration files, or when resolving DIDs within the consortium.
 
-In each mechanism, the resolver has the option to cache verified configuration files for a period of time (the cache lifetime being specified in the configuration), and in some of these, the resolver can automatically fetch and verify updates.
+##### Caching
+`"cache": {"max-age": [number in seconds]}`
 
-#### Automatic Bootstrapping
-Automatic bootstrapping relies on the assumption of trust in the owner of a web domain. In resolving a did:trustbloc DID, the resolver assumes the consortium domain in the DID is safe, and that the stakeholder domains within the consortium config are also safe and correct.
+This element specifies the length of time that a client should cache a copy of the consortium config.
 
-On a first connection by a resolver to the did:trustbloc method, the did method provides a mechanism for the resolver to establish trust in the consortium and stakeholders.
+##### Stakeholder Queries
+`""num-queries": [number of queries]`
 
-A resolver can begin by assuming trust in the consortium domain contained within a bloc DID. The config found there will inform the resolver about all the stakeholder organizations within the consortium. The resolver can trust the authenticity of these stakeholders based on their domain name registration, as well as their inclusion within the consortium config. 
+This element specifies the number of stakeholders that a client should query when verifying a consortium configuration - whether for bootstrapping or for updating.
 
-The resolver must verify at least M stakeholders, to retrieve their endpoints and confirm their signatures on the consortium config.
-
-For each stakeholder, the resolver fetches the stakeholder config from their domain. This includes a list of Sidetree endpoints managed by the stakeholder. With one of these, the resolver can resolve the stakeholder's DID to retrieve the stakeholder's DID doc. This DID Doc contains the key used to verify the signature on the stakeholder's config file, and one of the signatures on the consortium config file.
+### Bootstrapping Trust
+The Bloc DID method is designed to offer several mechanisms for a resolver to confirm that it can trust the stakeholders to provide the DID doc ledger service. 
```

Comment:
```suggestion
The TrustBloc DID method is designed to offer several mechanisms for a client to confirm that it can trust the stakeholders to provide the DID doc ledger service. 
```